app/components/post_draft/draft_input/draft_input.tsx

```diff
@@ -65,37 +68,76 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
             borderTopLeftRadius: 12,
             borderTopRightRadius: 12,
         },
+        actionsContainer: {
+            display: 'flex',
+            flexDirection: 'row',
+            justifyContent: 'space-between',
+            alignItems: 'center',
+            paddingBottom: Platform.select({
+                ios: 1,
+                android: 2,
+            }),
+        },
     };
 });
 
 export default function DraftInput({
-    testID,
+    addFiles,
+    canSend,
     channelId,
     currentUserId,
+    cursorPosition,
     files,
     maxMessageLength,
     rootId = '',
-    value,
-    uploadFileError,
     sendMessage,
-    canSend,
-    updateValue,
-    addFiles,
+    testID,
     updateCursorPosition,
-    cursorPosition,
     updatePostInputTop,
+    updateValue,
+    uploadFileError,
+    value,
     voiceMessageEnabled,
 }: Props) {
+    const [recording, setRecording] = useState(false);
     const theme = useTheme();
+    const style = getStyleSheet(theme);
 
     const handleLayout = useCallback((e: LayoutChangeEvent) => {
         updatePostInputTop(e.nativeEvent.layout.height);
     }, []);
 
-    const [recording, setRecording] = useState(false);
+    const onPresRecording = useCallback(() => {
+        setRecording(true);
+    }, []);
 
-    // Render
-    const style = getStyleSheet(theme);
+    const onCloseRecording = useCallback(() => {
+        setRecording(false);
+    }, []);
+
+    const getActionButton = useCallback(() => {
+        const sendActionTestID = `${testID}.send_action`;
+        const recordActionTestID = `${testID}.record_action`;
+
+        if (value.length === 0 && files.length === 0) { // add config condition to it
```

Comment:
You have the config already in place, is the prop `voiceMessageEnabled`.

Reply:
Thanks. Corrected